python/nano/src/bigdl/nano/pytorch/inference/optimizer.py

```diff
@@ -653,6 +658,31 @@ def load(path, model: Optional[nn.Module] = None):
         """
         return load_model(path, model)
 
+    @staticmethod
+    def to_multi_instance(model: nn.Module, num_processes: int) -> _MultiInstanceModel:
+        """
+        Transform a model to multi-instance inference model.
+        :param model: The model to transform.
+        :param num_processes: The number of processes which will be used.
+        :return: Model with multi-instance inference acceleration.
+        """
+        p_num = num_processes
+        mgr = mp.Manager()
+        send_queues = [mgr.Queue() for _ in range(p_num)]
+        recv_queues = [mgr.Queue() for _ in range(p_num)]
+        envs = schedule_processors(p_num)
+        ps = []
+        for i in range(p_num):
+            os.environ["KMP_AFFINITY"] = envs[i]['KMP_AFFINITY']
+            os.environ["OMP_NUM_THREADS"] = envs[i]['OMP_NUM_THREADS']
```

Comment:
shall we restore the env for the main process afterwards?

Reply:
yes, fixed it in <https://github.com/intel-analytics/BigDL/pull/6653>